controllers/controllers.js

```diff
@@ -21,4 +21,26 @@ exports.getArticleById=(req,res,next) => {
         return res.status(200).send({article});
     });
 
-};
\ No newline at end of file
+};
+
+exports.patchArticleById=(req,res,next) =>{
+    
+    const {article_id} =req.params;
+    const {inc_votes} = req.query;
+
+    if(Number.isNaN(+article_id)){
+        next({status:400, msg:'Invalid article_id'})
```

Comment:
Please use `Promise.reject` and let the error handlers handle this! Ideally you want your `next` inside your catch block.